sdk/README.md

```diff
@@ -4,26 +4,53 @@ There is an [SDK](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/)
 for `Kubeflow Pipeline` for end users to define end to end machine learning and data pipelines.
 The output of the KFP SDK compiler is YAML for [Argo](https://github.com/argoproj/argo).
 
-We are updating the `Compiler` of the KFP SDK to generate `Tekton` YAML. Please go through these steps to ensure you are setup properly to use the updated compiler.
+We are updating the `Compiler` of the KFP SDK to generate `Tekton` YAML. Please go through the steps below
+to ensure you are set up properly to use the KFP-Tekton compiler.
+
+
+## Table of Contents
+
+<!-- START of ToC generated by running ./tools/mdtoc.sh sdk/README.md -->
+
+  - [Project Prerequisites](#project-prerequisites)
+  - [Tested Pipelines](#tested-pipelines)
+  - [How to use the KFP-Tekton Compiler](#how-to-use-the-kfp-tekton-compiler)
+  - [Build Tekton from Master](#build-tekton-from-master)
+  - [Additional Features](#additional-features)
+    - [1. Compile Kubeflow Pipelines as a Tekton `PipelineRun`](#1.-compile-kubeflow-pipelines-as-a-tekton-`pipelinerun`)
+    - [2. Compile Kubeflow Pipelines with Artifacts Enabled](#2.-compile-kubeflow-pipelines-with-artifacts-enabled)
```

Comment:
```suggestion
    - [2. Compile Kubeflow Pipelines with Artifacts Enabled](#2-compile-kubeflow-pipelines-with-artifacts-enabled)
```

Reply:
I updated the `mdtoc.sh` script as well. Thanks @Tomcli